src/tree_store/page_store/mmap.rs

```diff
@@ -58,9 +82,9 @@ impl Mmap {
     }
 
     pub(crate) fn get_memory(&self, range: Range<usize>) -> &[u8] {
```

Comment:
I wonder if `get_memory` should also be unsafe, since it can't be called with a range that overlaps with a current borrow from `get_memory_mut`.

Reply:
Ah yes, that does seem better